compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -2002,6 +2017,14 @@ static LogicalResult setRootConfig(IREE::GPU::TargetAttr target,
         return setUKernelConfig(entryPointFn, ukernelOp);
       })
       .Default([&](auto op) {
+        if (clGPUEnableTileAndFuse) {
+          // Default lowering config using tile and fuse instead.
+          if (succeeded(IREE::GPU::setTileAndFuseLoweringConfig(
+                  target, entryPointFn, computeOp))) {
+            LDBG("Tile and fuse default config");
+            return success();
+          }
+        }
```

Comment:
This should be behind the new `clGPUTestTileAndFuse` right?

Reply:
the `clGPUTestTileAndFuse` is just a testing flag for now so that I can override all other pipelines. If you have another suggestion about how to organize the flags I am open to it because two flags is certainly unwieldy.

Reply:
This is overriding the default lowering pipeline to use tile and fuse. Shouldn't that be behind the testing flag? My understanding was that we would start with adding SIMT matmuls, and then add more to the TileAndFuse pipeline.

Reply:
I went ahead and removed this part because it doesn't make too much sense right now and split the previous flag and the new flag into two distinct flags with better naming: `--iree-codegen-llvmgpu-test-tile-and-fuse-{matmul|vectorize}`.